client/state/current-user/reducer.js

```diff
@@ -57,7 +58,10 @@ export const capabilities = createReducer( {}, {
 	}
 }, capabilitiesSchema );
 
+export { actionLog };
```

Comment:
I'd suggest moving tests to `state/current-user/action-log/test/reducer.js` to avoid exporting here.